### PR TITLE
Format spring package.

### PR DESCRIPTION
**Pull request by asashour**
The current code sometimes uses 2-space indentation and 4-space one.

I think it makes sense to format the spring package (for now) for 4-space indentation, as a preparation to work on a PR for that package.

Please let me know if 2-space one is preferred.

### CODE REVIEW

**Review by MalloD12 — APPROVED** [review]
**Approved**.

Formatting changes look good to me. I removed a deprecated set method and its related get method as well, since they were not used and there was a codeQL scan issue reported (due to an unused argument).

No builds or test checks issues reported.

File: liquibase-standard/src/main/java/liquibase/integration/spring/MultiTenantSpringLiquibase.java [hunk 1]
```diff
@@ -76,252 +79,253 @@ public class MultiTenantSpringLiquibase implements InitializingBean, ResourceLoa
...
+        liquibase.setDefaultSchema(defaultSchema);
+        liquibase.setLiquibaseSchema(liquibaseSchema);
+        liquibase.setLiquibaseTablespace(liquibaseTablespace);
+        liquibase.setDatabaseChangeLogTable(databaseChangeLogTable);
+        liquibase.setDatabaseChangeLogLockTable(databaseChangeLogLockTable);
+        return liquibase;
+    }
+
+
+    public String getJndiBase() {
```
Comment (filipelautert) [inline liquibase-standard/src/main/java/liquibase/integration/spring/MultiTenantSpringLiquibase.java:183]: As we are reviewing this class those methods could be replaced by lombok's @Setter and @Getter .

**Review by filipelautert — CHANGES_REQUESTED** [review]

File: liquibase-standard/src/main/java/liquibase/integration/spring/MultiTenantSpringLiquibase.java (hunk 1 above)
Comment (asashour) [reply liquibase-standard/src/main/java/liquibase/integration/spring/MultiTenantSpringLiquibase.java]: Done, the setters and getters which have JavaDoc are preserved.

**Review by filipelautert — APPROVED** [review]

File: liquibase-standard/src/main/java/liquibase/integration/spring/MultiTenantSpringLiquibase.java (hunk 1 above)
Comment (filipelautert) [reply liquibase-standard/src/main/java/liquibase/integration/spring/MultiTenantSpringLiquibase.java]: Thanks @asashour !

**Pull request merged**